hcl2template/addrs/plugin.go

```diff
@@ -0,0 +1,238 @@
+package addrs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2"
+	"golang.org/x/net/idna"
+)
+
+// Plugin encapsulates a single plugin type.
+type Plugin struct {
+	Type      string
+	Namespace string
+	Hostname  string
+}
+
+func (p Plugin) RealRelativePath() string {
+	ns := DefaultPluginNamespace
+	if p.Namespace != "" {
+		ns = p.Namespace
+	}
+	return "/" + ns + "/packer-plugin-" + p.Type
```

Comment:
it feels a little weird to me to have a relative path prefixed with /

Reply:
Oh, good catch, I actually remember wanting to remove that !

```suggestion
	return ns + "/packer-plugin-" + p.Type
```